dragonchain/lib/dto/bnb.py

```diff
@@ -203,7 +203,7 @@ def check_balance(self, symbol: str = "BNB") -> int:
         # cannot check HTTP status codes, errors will return 200 :
         if response_json.get("error") is not None:
             if "interface is nil, not types.NamedAccount" in response_json["error"]["data"] and response.status_code == 500:
-                _log.warning(f"[BINANCE] Non 200 response from Binance node:")
+                _log.warning("[BINANCE] Non 200 response from Binance node:")
```

Comment:
whats the deal here with the f flag?

Reply:
linting doesn't allow you to use this type of string formatting unless you are actually printing a variable in the string itself